cypress/support/component-index.html

```diff
@@ -5,8 +5,11 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
-  <script src="https://accounts.google.com/gsi/client" async defer id="gsiClient"></script>
```

Comment:
Another drive by fix - we removed "google sign in" in https://github.com/DataBiosphere/duos-ui/pull/2667